### PR TITLE
Project status should include some pod information

### PR DESCRIPTION
**Pull request by smarterclayton**
Including the desired state annotations for rolling update

@fabianofranz review please

### CODE REVIEW

File: pkg/cmd/cli/describe/projectstatus.go [hunk 1]
```diff
@@ -387,20 +387,60 @@ func describeDeployments(node *graph.DeploymentConfigNode, count int) []string {
...
+			change = fmt.Sprintf(" growing to %d", changing)
+		}
+	}
+	return fmt.Sprintf(" - %s%s", s, change)
+}
+
+func describeDeploymentPodSummary(deploy *kapi.ReplicationController, showEmpty bool) string {
+	actual, requested := deploy.Status.Replicas, deploy.Spec.Replicas
+	if actual == requested {
+		if actual == 0 {
```
Comment (fabianofranz) [inline pkg/cmd/cli/describe/projectstatus.go]: `switch`

File: pkg/cmd/cli/describe/projectstatus_test.go [hunk 2]
```diff
@@ -137,6 +137,8 @@ func TestProjectStatus(t *testing.T) {
 				"database deploys",
 				"frontend deploys",
 				"with docker.io/openshift/ruby-20-centos7:latest",
+				"#2 deployment failed less than a second ago: unable to contact server - 0/1 pods",
+				"#2 deployment running for 7 seconds - 2/1 pods",
 				"#1 deployed 8 seconds ago",
```
Comment (fabianofranz) [inline pkg/cmd/cli/describe/projectstatus_test.go]: I get that you want the ID as the first thing in line, but this seam more readable:

```
deployment #2 running for 7 seconds - 2/1 pods
```

**Comment by fabianofranz** [comment]
I found the login around the strings concatenation and punctuation a little hard to follow, but I guess it's still better than a Go template. So make sure you have tests for all scenarios (`showEmpty`, empty `reason`, etc). Otherwise LGTM.

**Comment by fabianofranz** [comment]
s/login/logic

File: pkg/cmd/cli/describe/projectstatus_test.go (hunk 2 above)
Comment (smarterclayton) [inline pkg/cmd/cli/describe/projectstatus_test.go]: Not sure what you're asking

----- Original Message -----

Comment (fabianofranz) [inline pkg/cmd/cli/describe/projectstatus_test.go]: Splitting `#n deployment` => `deployment #n` seems more readable, unless we want #n as the first thing in line. 

But I'm actually referring to line https://github.com/openshift/origin/pull/2423/files#diff-c0c5022b465088097f483637ede5dcdaR401 and the following lines.  :)

Comment (smarterclayton) [inline pkg/cmd/cli/describe/projectstatus_test.go]: I'm mostly concerned about scanability.  The goal is to find the # as fast as possible visually.  For right now, that's best done on the left margin of the text.

File: pkg/cmd/cli/describe/projectstatus.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/cmd/cli/describe/projectstatus.go]: Fixed

----- Original Message -----

**Comment by smarterclayton** [comment]
[merge]
